lib/interfaces/swagger-document-options.interface.ts

```diff
@@ -1,3 +1,15 @@
+export type OperationIdFactory =
```

Comment:
Why union here?

Reply:
Oh, not sure how that slipped in. Removed.